playbooks/robusta_playbooks/playbooks_utils.py

```diff
@@ -0,0 +1,175 @@
+from robusta.api import *
```

Comment:
Playbooks_utils might be used by other playbooks packages as well, I assume.
So I think it's better not to include it in the default playbooks package. (Other packages don't have access to it)
I would add it as a part of the core code, and add it to robusta.api
This way, it can be imported and used by any playbooks package

Reply:
I'll create my own prometheus chart utils